src/cpp/ext/otel/otel_plugin.h

```diff
@@ -366,6 +365,17 @@ class OpenTelemetryPlugin : public grpc_core::StatsPlugin {
   };
 
   // StatsPlugin:
```

Comment:
Maybe move this comment to line 379 which marks the start of the StatsPlugin's methods.

Reply:
I see, I didn't realize what you meant by StatsPlugin's methods